pkg/ingress/v1/types.go

```diff
@@ -0,0 +1,31 @@
+package v1
+
+import (
+	"context"
+
+	ingresscommon "github.com/kanopy-platform/argoslower/pkg/ingress"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+// EventSourceIngressConfig provides the information needed for rendering
+// ingress resources mapped to the service of an argo event source.
+// it is ingress provider agnostic.
+type EventSourceIngressConfig struct {
+	Ipg            IPGetter
+	Es             types.NamespacedName
```

Comment:
I think this should have a more descriptive name as an exported value.

Reply:
Refactored to be more explicit